internal/constants.go

```diff
@@ -16,5 +16,7 @@ package internal
 
 const CommandServiceKey            = "edgex-core-command"
 const CoreDataServiceKey           = "edgex-core-data"
+const ExportClientServiceKey       = "edgex-export-client"
+const ExportDistroServiceKey       = "edgex-export-distro"
 const MetaDataServiceKey           = "edgex-core-metadata"
```

Comment:
Maybe the layer name can be added in "CommandServiceKey" and "MetaDataServiceKey", so it will be "CoreCommandServiceKey" and "CoreMetaDataServiceKey". This is how it's done with other service keys.

But anyway, looks good for me.